src/pages/SignUpPageNew/index.js

```diff
@@ -19,12 +19,21 @@ class SignUpPage extends React.Component {
     this.state = { ...INITIAL_STATE };
   }
 
-  handleSubmit = (values, setSubmitting) => {
+  handleSubmit = async (values, setSubmitting) => {
     const signUpSubmission = {
       ...values,
     };
 
-    console.log(JSON.stringify(signUpSubmission));
+    const response = await fetch(SIGN_UP_URL, {
```

Comment:
throw this behind something in services - frontend pages should not know anything about how we're signing up